### DISCUSSION TITLE
[求助]谁能告诉我这是什么↓

### DISCUSSION THREAD

**Discussion by CrayHuan233** [share121/inter-knot#4559] [原神]
今天跟影月月弄探索度发现的，因不知为何物，故发此贴，望周知
![default](https://github.com/user-attachments/assets/1fe60aff-ce64-4521-a102-007eca709929)

**Comment by happyfucking** [comment]
每个部落各有三个，然后就能拼成三个环，然后到每个部落对应的地方能拿祝圣之霜，还有30个金色圣遗物

**Reply by CrayHuan233** [reply]
感谢

**Comment by LLleu** [comment]
![20240909221617](https://github.com/user-attachments/assets/e853c417-7f99-448b-933a-01cfe9206b95)
听说这玩意珍贵华丽随机出，那我连开宝箱都吃保底啦

**Reply by ChenYanxuan579** [reply]
玛拉妮可爱捏🥺

**Comment by mengsiliulian** [comment]
没用，建议不管（doge）

**Comment by SWYH9527** [comment]
这是可以逆天改命的东西，等时机到了就能用了
